Dockerfile-alpine

```diff
@@ -18,6 +18,8 @@ RUN apk add --no-cache \
         ca-certificates \
         clang \
         llvm \
+        lz4-dev \
+        zstd-dev \
```

Comment:
I wonder why we didn't hit to that issue when building nightly docker images ?

Reply:
also, should we add those two to `Dockerfile` too ?

Reply:
alpine has the smallest set of all packages compared to other base images. So it means that others already include that packages. Or we need to add these packages if they are needed on runtime . So after testing we may add these packages to other docker files

Reply:
I understood the reason. This is because, we directly install citus from packages in other docker images but we build citus from source for alpine, so we need those dependencies.

But I'm still wondering this:
>I wonder why we didn't hit to that issue when building nightly docker images ?